crime-applications-adaptor/src/main/resources/schemas/criminalapplicationsdatastore/general/offence.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
we are not using this json , please verify

Reply:
Correct, there are a few others under src/main/resources/schemas/criminalapplicationsdatastore which we also don't currently use. I was considering one of the following approaches (I don't have a strong feeling either way) so happy to go with your opinion?

1. Copy all schemas from https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/tree/main/schemas/1.0 to `src/main/resources/schemas/criminalapplicationsdatastore` so we have an exact one to one likeness with the schema 'source of truth'
2. Copy only the schema files we need/use currently from https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/tree/main/schemas/1.0 to `src/main/resources/schemas/criminalapplicationsdatastore` in order to try and simplify our microservice code

Reply:
Please only copy the schema files we need otherwise it might cause confusion. thanks

Reply:
sure thing, will do

Reply:
done